security/advancedtls/advancedtls_test.go

```diff
@@ -679,3 +681,86 @@ func TestOptionsConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCertificateSNI(t *testing.T) {
+	// Define const variables needed to set up ClientHelloInfo for testing.
+	const (
+		tlsEcdheRsaWith3desEdeCbcSha uint16 = 0xc012
+	)
+	const (
+		CurveP256 tls.CurveID = 23
+	)
+	const (
+		pointFormatUncompressed uint8 = 0
+	)
+	const (
+		VersionTLS10 uint16 = 0x0301
+	)
```

Comment:
Can we remove the const declarations here?

Reply:
I removed these three const declarations and instead called the corresponding exported const variable from tls library directly.
```	
        const (
		tlsEcdheRsaWith3desEdeCbcSha uint16 = 0xc012
	)
	const (
		CurveP256 tls.CurveID = 23
	)
	const (
		VersionTLS10 uint16 = 0x0301
	)
```
However, for `pointFormatUncompressed`, it's an unexported const in tls library so I have to re-declare it. A reference of the code can be found [here](https://github.com/golang/go/blob/cd10f5f632983e63b156478abdfa0bd57e93898c/src/crypto/tls/common.go#L145-L147).